packages/plugin-hardhat/package.json

```diff
@@ -30,16 +30,20 @@
     "fgbg": "^0.1.4",
     "hardhat": "^2.0.2",
     "promisified": "^0.5.0",
-    "rimraf": "^5.0.0"
+    "proxyquire": "^2.1.3",
+    "rimraf": "^5.0.0",
+    "sinon": "^15.0.0"
   },
   "dependencies": {
-    "@openzeppelin/defender-base-client": "^1.47.0",
-    "@openzeppelin/platform-deploy-client": "^0.9.0",
+    "@openzeppelin/defender-admin-client": "^1.48.0",
+    "@openzeppelin/defender-base-client": "^1.48.0",
+    "@openzeppelin/platform-deploy-client": "^0.10.0",
```

Comment:
I remember now why we had separate packages. We didn't want to install these dependencies for users who didn't need them.

We already abandoned this when we added Platform support so this is not new in this PR.

We should monitor that the Defender packages don't grow too large or have too many dependencies and ask the team to be mindful about that.